test-scripts/local-is-deployment/test-config.yaml

```diff
@@ -16,8 +16,6 @@
 
 # TestGrid Test Configuration Parameters
 productName: "IS"
```

Comment:
Let's add the full product name here (`wso2-is-5.4.0-LTS`)

Reply:
This is fixed.

Reply:
*wso2is :)